src/components/filters/polygon-filter.tsx

```diff
@@ -30,43 +30,41 @@ const isAlreadySelected = (selectedLayers: Layer[], layerId: string) =>
   selectedLayers.findIndex(l => l.id === layerId) === -1;
 
 function PolygonFilterFactory() {
-  const PolygonFilter: React.FC<PolygonFilterProps> = React.memo(
-    ({filter, layers, setLayers}) => {
-      const setNewLayers = useCallback(
-        newLayers => {
-          return setLayers(newLayers.map((l: Layer) => l.id));
-        },
-        [setLayers]
-      );
-
-      const selectedLayers = useMemo(() => layers.filter(l => filter.layerId?.includes(l.id)), [
-        filter,
-        layers
-      ]);
+  const PolygonFilter: React.FC<PolygonFilterProps> = React.memo(({filter, layers, setLayers}) => {
```

Comment:
why are these styling changed? is there a prettier version upgrade?

Reply:
No, I haven't upgraded prettier or it's configuration... But I've checked now and prettier has changed noting so should I return the previous format there manually?